doc/index.md

```diff
@@ -136,6 +136,7 @@ Development and maintenance of sttp tapir is sponsored by [SoftwareMill](https:/
    :caption: Server interpreters
 
    server/akkahttp
+   server/aws
```

Comment:
it's a bit ironic that's a "serverless" interpreter is listed under "server interpreters", but I guess that's the proper place ;)